test/unit/services/helpers/test-caseload-helper.js

```diff
@@ -19,49 +18,223 @@ const CASELOAD = {
   a: 0
 }
 
-const CUSTODY_CASELOAD_0 = Object.assign({}, CASELOAD, { caseType: caseType.CUSTODY })
-const COMMUNITY_CASELOAD_0 = Object.assign({}, CASELOAD, { totalCases: 6, caseType: caseType.COMMUNITY })
-const LICENSE_CASELOAD_0 = Object.assign({}, CASELOAD, { totalCases: 9, caseType: caseType.LICENSE })
-const OVERALL_CASELOAD_0 = Object.assign({}, CASELOAD, { totalCases: 18, untiered: 3, d2: 3, c1: 3, b1: 3 })
+// linkId = 2, grade = PO
+const CUSTODY_CASELOAD_PO_0 = Object.assign({}, CASELOAD, { linkId: 2, caseType: caseType.CUSTODY })
```

Comment:
Maybe worth taking this test data out into a separate helper file.

Reply:
done